pkg/apis/tensorflow/helper/helpers.go

```diff
@@ -110,10 +110,12 @@ func Cleanup(c *tfv1.TFJobSpec) {
 	// We should have default container images so user doesn't have to provide these.
 }
 
+// CRDName : method returns custom resource definition kind and group.
 func CRDName() string {
 	return fmt.Sprintf("%s.%s", tfv1.CRDKindPlural, tfv1.CRDGroup)
 }
 
+// scalingReason : method returns the reason for scaling the cluster size
```

Comment:
e.g. here it would be
"scalingReason returns the reason for scaling the cluster size."